### PR TITLE
feat: created modal and toast

### PR DESCRIPTION
**Pull request by lakshaychopra**

### CODE REVIEW

File: components/toast/toast.css [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    width: 20rem;
+    justify-content: center;
+    border-radius: 5px;
+    gap: 1rem;
+    position: relative;
+}
+
+.toast-success {
+    border: 2px solid #81c784;
+    color: #5dab63;
```
Comment (someshtiwarii) [inline components/toast/toast.css:14]: can use color variables

**Review by someshtiwarii — APPROVED** [review]
just add color variables, rest looks good to merge

File: components/lists/list.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+.list-item {
+    padding: 0.5rem;
+}
+
+.spaced-list .list-item {
+    padding: 1rem;
+}
+
+.stacked-list .list-item {
+    padding: 0rem;
```
Comment (Sen-442b) [inline components/lists/list.css:39]: Suggestion 

- You don't need to use any kind of absolute or relative unit if the value is 0;

File: components/text-utilities/text-utilities.css [hunk 3]
```diff
@@ -0,0 +1,42 @@
+h1,
+.h1 {
```
Comment (Sen-442b) [inline components/text-utilities/text-utilities.css:2]: Any specific reason to give same stylings to both the element and its relevant class?

**Review by Sen-442b — APPROVED** [review]
Other things look fine to me

**Pull request merged**